### PR TITLE
Resource bundle

### PR DESCRIPTION
**Pull request by obj-p**
# Description
###### Jira Issue: [IOS-183](https://stockx-services.atlassian.net/browse/IOS-183)

This sets up Armchair to use `resource_bundle` property of the podspec. It does also update Armchair to 4.2.

### CODE REVIEW

File: Armchair.xcodeproj/xcshareddata/xcschemes/Armchair.xcscheme [hunk 1]
```diff
@@ -28,16 +28,6 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
-         <TestableReference
-            skipped = "NO">
-            <BuildableReference
```
Comment (LaurentShala) [inline Armchair.xcodeproj/xcshareddata/xcschemes/Armchair.xcscheme:33]: Can you explain what is happening here? Was this automatic or did you manually remove these?

File: Armchair.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -720,7 +718,7 @@
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 4.2;
```
Comment (etlasky) [inline Armchair.xcodeproj/project.pbxproj:721]: Do we need to update the swift version? Is this possibly problematic?

File: Armchair.xcodeproj/xcshareddata/xcschemes/Armchair.xcscheme (hunk 1 above)
Comment (obj-p) [reply Armchair.xcodeproj/xcshareddata/xcschemes/Armchair.xcscheme:33]: They had a phantom test target left in their scheme. We don't really need this change one way or another, however, I plan to open a PR back to upstream Urban when I clean it up more. For now, these changes accomplish what we need and this is just extra

Comment (LaurentShala) [reply Armchair.xcodeproj/xcshareddata/xcschemes/Armchair.xcscheme:33]: Gotcha! Thanks for the explanation!

File: Armchair.xcodeproj/project.pbxproj (hunk 2 above)
Comment (obj-p) [reply Armchair.xcodeproj/project.pbxproj:721]: Their project actually doesn't build without bumping. And their podspec specifies 4.2

**Review by LaurentShala — APPROVED** [review]

File: Armchair.xcodeproj/project.pbxproj (hunk 2 above)
Comment (etlasky) [reply Armchair.xcodeproj/project.pbxproj:721]: Weird... But sounds good!

**Review by etlasky — APPROVED** [review]

**Pull request merged**